subxt/src/book/usage/blocks.rs

```diff
@@ -24,14 +24,14 @@
 //! - [events](crate::blocks::Block::events())
 //! - [runtime APIs](crate::blocks::Block::runtime_api())
 //!
+//! Aside from these links to other Subxt APIs, the main thing that we can do here is iterate over and
+//! decode the extrinsics in a block body.
```

Comment:
I tried to keep referring to anything in a block body as an extrinsic (since they could be transactions or inherents), but anything we are submitting as a transaction (mainly because we call the module `tx` etc, so wanted docs to align with this naming)